src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -130,8 +137,8 @@ export default class DeclarativeTableComponent extends ContainerBase<any> implem
 		}
 	}
 
-	private onCellDataChanged(newValue: any, rowIdx: number, colIdx: number): void {
-		this.data[rowIdx][colIdx] = newValue;
+	private onCellDataChanged(newValue: string | number | boolean | any, rowIdx: number, colIdx: number): void {
```

Comment:
can we get rid of `any` here?

Reply:
Unfortunately for back-compat I can't get rid of any :( We really should avoid designing APIs with any and definitely should not promote them to the stable API.

Reply:
I've deprecated the any fields, hopefully in a few months we can get rid of them.